src/aws/acm-wrapper.ts

```diff
@@ -52,50 +53,42 @@ class ACMWrapper {
     /**
      * * Gets Certificate ARN that most closely matches Cert ARN and not expired
      */
-    private async getCertArnByCertName(certificates, certName): Promise<string> {
-        // note: we only check DomainName, but a future enhancement could
-        // be to also check SubjectAlternativeNames
-        const matches = certificates.filter((certificate) => (certificate.DomainName === certName));
-        for (const certificate of matches) {
-            const certificateArn = certificate.CertificateArn;
-            const details = await throttledCall(
-                this.acm,
-                "describeCertificate",
-                {CertificateArn: certificateArn},
-            );
-            const currNotAfter = details.Certificate.NotAfter;
-            if (Date.now() < currNotAfter) {
-                Globals.logInfo(
-                    `Selecting cert with ARN=${certificateArn} with future expiry (${currNotAfter.toISOString()})`
-                );
-                return certificateArn;
-            }
-            Globals.logInfo(
-                `Ignoring cert with ARN=${certificateArn} that is expired (${currNotAfter.toISOString()})`
-            );
+    private getCertArnByCertName(certificates, certName): string {
+        const found = certificates.find((c) => c.DomainName === certName);
+        if (found) {
+          return found.CertificateArn;
         }
+        return null;
```

Comment:
I removed the code around checking for expired certs because we don't need it with the cert status filter on line 6. Then I refactored what was left of the code.